addons/website/static/src/scss/website.scss

```diff
@@ -416,7 +416,7 @@ font[class*='bg-'] {
 
 // Probably outdated
 // Disable fixed height
-@media (max-width: 400px) {
+@include media-breakpoint-down(sm) {
```

Comment:
:notebook: Note: this should be adapted in V16 with Bootstrap 5 because of [this](https://github.com/odoo/odoo/commit/9b5c9484fe0bf2302f4269defebef18aff458dca)